core/app/models/concerns/spree/user_address.rb

```diff
@@ -30,6 +32,21 @@ def persist_order_address(order)
           update(ship_address_id: s_address.id)
         end
       end
+
+      private
+
+      BELONGS_TO_OTHER_USER = 'belongs to other user'
+      ADDRESS_ASSOCIATED_WITH_COMPLETED_ORDER = 'associated with completed order'
```

Comment:
@KacperMekarski this needs to be taken from translations

Reply:
done